test/torchaudio_unittest/functional/functional_cpu_test.py

```diff
@@ -121,16 +121,18 @@ def test_DB_to_amplitude(self):
         torch.testing.assert_allclose(x2, spec, atol=5e-5, rtol=1e-5)
 
 
-@pytest.mark.parametrize('complex_tensor', [
-    torch.randn(1, 2, 1025, 400, 2),
-    torch.randn(1025, 400, 2)
-])
-@pytest.mark.parametrize('power', [1, 2, 0.7])
-def test_complex_norm(complex_tensor, power):
-    expected_norm_tensor = complex_tensor.pow(2).sum(-1).pow(power / 2)
-    norm_tensor = F.complex_norm(complex_tensor, power)
-
-    torch.testing.assert_allclose(norm_tensor, expected_norm_tensor, atol=1e-5, rtol=1e-5)
+class TestComplexNorm(common_utils.TorchaudioTestCase):
+
+    @parameterized.expand([
+        (torch.randn(1, 2, 1025, 400, 2),  1),
+        (torch.randn(1025, 400, 2), 2)
+    ])
+    def test_complex_norm(self, complex_tensor, power):
```

Comment:
Performing `torch.randn` in decorator makes it difficult to control the randomness, so instead can you parameterize the shape?

```
def test_complex_norm(self, shape, power):
    torch.manual_seed(0)
    complex_tensor = torch.randn(*shape)
```